File: tkpdseller/src/main/java/com/tokopedia/seller/common/adapter/viewholders/LoadingViewholder.java
package com.tokopedia.seller.common.adapter.viewholders;

import android.support.annotation.LayoutRes;
import android.view.View;

import com.tokopedia.seller.common.adapter.model.LoadingModel;


/**
 * @author Kulomady on 1/25/17.
 */

public class LoadingViewholder extends AbstractViewHolder<LoadingModel> {
    @LayoutRes
    public final static int LAYOUT = com.tokopedia.core.R.layout.loading_layout;

    public LoadingViewholder(View itemView) {
        super(itemView);
    }

    @Override
    public void bind(LoadingModel element) {

    }
}

File: features/discovery/home_wishlist/src/main/java/com/tokopedia/home_wishlist/view/viewholder/LoadingViewHolder.kt
package com.tokopedia.home_wishlist.view.viewholder

import android.view.View
import com.tokopedia.home_wishlist.R
import com.tokopedia.home_wishlist.model.datamodel.LoadingDataModel
import com.tokopedia.smart_recycler_helper.SmartAbstractViewHolder
import com.tokopedia.smart_recycler_helper.SmartListener

class LoadingViewHolder(view: View) : SmartAbstractViewHolder<LoadingDataModel>(view){
    override fun bind(element: LoadingDataModel, listener: SmartListener) {}

    companion object{
        val LAYOUT = R.layout.layout_loading_list
    }
}